# HG changeset patch
# User andrew
# Date 1345732943 -3600
# Node ID baf30df50ce385c2456ef5eb09683d17699b5481
# Parent  2c6933c5106b81a8578b70996fe5b735fb3adb60
7192804: Build should not install jvisualvm man page for OpenJDK
Summary: OpenJDK builds don't ship VisualVM so shouldn't ship its man page either.
Reviewed-by: dholmes

diff --git a/make/common/Release.gmk b/make/common/Release.gmk
--- jdk8/jdk/make/common/Release.gmk
+++ jdk8/jdk/make/common/Release.gmk
@@ -156,7 +156,6 @@
         jstack.1           \
         jstat.1            \
         jstatd.1           \
-        jvisualvm.1        \
 	native2ascii.1     \
 	rmic.1             \
         schemagen.1        \
@@ -165,6 +164,10 @@
         wsimport.1         \
         xjc.1
 
+ifndef OPENJDK
+  JDK_MAN_PAGES += jvisualvm.1
+endif
+
 ifeq ($(PLATFORM), solaris)
   MANBASEDIRS=$(JDK_TOPDIR)/src/solaris/doc $(IMPORTDOCDIR)
   MAN1SUBDIR=sun/man/man1
diff --git a/makefiles/Images.gmk b/makefiles/Images.gmk
--- jdk8/jdk/makefiles/Images.gmk
+++ jdk8/jdk/makefiles/Images.gmk
@@ -237,7 +237,6 @@
         jstack.1           \
         jstat.1            \
         jstatd.1           \
-	jvisualvm.1        \
 	native2ascii.1     \
 	rmic.1             \
         schemagen.1        \
@@ -246,6 +245,10 @@
         wsimport.1         \
         xjc.1
 
+ifndef OPENJDK
+  JDK_MAN_PAGES += jvisualvm.1
+endif
+
 ifeq ($(OPENJDK_TARGET_OS), linux)
     MAN_SRC_DIR:=$(JDK_TOPDIR)/src/linux/doc
     MAN1_SUBDIR:=man
